crates/host/src/wasmbus/mod.rs

```diff
@@ -2462,6 +2477,7 @@ impl Host {
                 image_reference: actor_ref.to_string(),
                 actor_claims: Arc::clone(&self.actor_claims),
                 provider_claims: Arc::clone(&self.provider_claims),
+                metrics: self.metrics.clone(),
```

Comment:
should we wrap this in an Arc?

Reply:
Yep, I think that's probably the right move here. I've made it into one.